### ISSUE TITLE
ElementsCollection annotated with @FindBy is not initialized

### ISSUE THREAD

**Issue by rishaselfing** [codeborne/selenide#186]
When using Page Object pattern PageFactory does not decorate ElementsCollection fields, so I'm getting NullPointerException. You can reproduce it on your own Google classic page object example.

**Comment by asolntsev** [comment]
Thank you for reporting, I will investigate it asap.

**Comment by asolntsev** [comment]
Hi @rishaselfing !
In the "classic_page_object" folder, there is a test using pure Selenium code. I mean, class `org.selenide.examples.google.classic_page_object.GoogleTest` does **not** use Selenide. It uses selenium built-in factory `org.openqa.selenium.support.PageFactory`. It's not supposed to support Selenide classes. 

Thought, you are right, Selenide page factory actually cannot inject ElementsCollection fields.
I have fixed it in Selenide 2.20 (it's not released yet). I have also added such an example to Google example:
https://github.com/selenide-examples/google/tree/master/test/org/selenide/examples/google/selenide_page_object_with_fields
